src/main/java/com/thirdcc/webapp/lib/wrapper/GCPCloudStorage.java

```diff
@@ -0,0 +1,27 @@
+package com.thirdcc.webapp.lib.wrapper;
+
+import com.google.cloud.storage.*;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.Objects;
+
+@Service
+public class GCPCloudStorage implements CloudStorage {
+    private Storage storage = StorageOptions.newBuilder().setProjectId("ccclubmanagement").build().getService();
+
+    @Override
+    public String store(MultipartFile multipartFile) throws IOException {
+        BlobId blobId = BlobId.of("cc-club-management-bucket-1", Objects.requireNonNull(multipartFile.getOriginalFilename()));
```

Comment:
Maybe we can make the bucket dynamic also?
For example we can have bucket for UserProfileImage, Event-1--Transaction

Can make a Issue for enhancement la
No need fix immediately

Reply:
ok